src/Build/Resources/Strings.resx

```diff
@@ -1508,6 +1508,9 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
   <data name="OM_MatchingProjectAlreadyInCollection">
     <value>An equivalent project (a project with the same global properties and tools version) is already present in the project collection, with the path "{0}". To load an equivalent into this project collection, unload this project first.</value>
   </data>
+  <data name="OM_MatchOnMetadataIsRestrictedToOnlyOneReferencedItem">
+    <value>Removing with MatchOnMetadata only supports one referenced item and one metadata value.</value>
```

Comment:
We decided to limit the scope to 1 for now. We had some discussion about what the behavior should be if there are 2 metadata values and there is a match on 1 or 0 but not both. Since there wasn't a clear winner in terms of the right behavior in some edge cases, we're limiting it to 1 in this PR.

Reply:
Great. Can you update the PR description to make future archaeology easier?